enabler/index.html

```diff
@@ -65,19 +65,30 @@ <h1 class="h3 mb-3 font-weight-normal">Dev Site</h1>
                   console.log('onComplete', { isSuccess, paymentReference });
                 },
               });
-        
+
               enabler.createComponentBuilder('card')
               .then(builder => {
                 const paymentElement = builder.build({
                   config: {},
                 });
                 paymentElement.mount('#card-component')
               })
+        
+              // Possible options are: `klarna_paynow` (Pay Now), `klarna` (Pay Later), `klarna_account` (Pay Over Time)
+              enabler.createComponentBuilder('klarna_paynow')
+              .then(builder => {
+                const paymentElement = builder.build({
+                  showPayButton: true // Must be set to true for Adyen/Klarna to render the redirect button in the enabler.
+                });
+                paymentElement.mount('#klarna-component')
+              })
             });
       
           </script>
       
           <div id="card-component"></div>
+          <hr>
```

Comment:
is this horizontal rule visible somehow? wonder if it's necessary

Reply:
Yes it's visible. (this file is only used for development locally) I used it to distinquish a tiny bit better where one ends and another one starts. See the comparison below.

![image](https://github.com/commercetools/connect-payment-integration-adyen/assets/107549011/2f7f5f64-e534-4891-ac8f-84e9f4879f23)

![image](https://github.com/commercetools/connect-payment-integration-adyen/assets/107549011/060b8037-162b-4207-8a6d-97facc0fc17e)

Reply:
> this file is only used for development locally
perfect then 😁